Cargo.toml

```diff
@@ -74,7 +74,7 @@ impl-codec = { path = "./primitive-types/impls/codec", default-features = false
 impl-num-traits = { path = "./primitive-types/impls/num-traits", default-features = false }
 impl-rlp = { path = "./primitive-types/impls/rlp", default-features = false }
 impl-serde = { path = "./primitive-types/impls/serde", default-features = false }
-kvdb = { path = "./kvdb" }
+kvdb = { path = "./kvdb", version = "0.13" }
```

Comment:
Why others do not need version? E.g. `ethereum-types = { path = "./ethereum-types" }`

Reply:
Probably we didn't need to publish a package that depends on it lately. I think it's best to fix each dependency when we hit the issue. If I try to fix everything in this PR, 'm afraid I might miss something